client/app/account/user/agreements/details/user-agreements-details.html

```diff
@@ -2,18 +2,13 @@
      class="module-useraccount-sections-agreements-container">
 
     <div class="page-header">
-        <h1 data-translate="user_agreement_details_title"></h1>
+        <h1 data-translate="user_agreements_title"></h1>
     </div>
 
     <div class="tab-content">
-        <a class="d-inline-block mb-5"
-           href="#/useraccount/agreements"
-           title="{{ 'user_agreement_details_return' | translate }}">
-            <span class="fa fa-chevron-left mr-2"
-                  aria-hidden="true">
-            </span>
-            <span data-translate="user_agreement_details_return"></span>
-        </a>
+        <oui-back-button data-href="{{ ctrl.$state.href('^') }}"
```

Comment:
More appropriate in that way:
```html
<div class="oui-back-button">
    <a class="oui-button oui-button_icon-alone"
       data-ng-attr-title="{{ :: 'user_agreement_details_title' | translate }}"
       data-ui-sref="app.account.service.useraccount.agreements">
        <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
    </a>
    <h2 class="oui-back-button_title oui-header_2"
        data-translate="user_agreement_details_title">
    </h2>
</div>
```

Reply:
This would not utilise the ui-kit, the ui-kit give us a chance to not define style manually

Reply:
@JDownloader Pay attention, snippet use exactly the UI Kit :).

Reply:
See https://master.ui-kit.ovh/#!/oui-angular/back-button, i'm not using the Style version since the dedicated manager is an Angular App

Reply:
@antleblanc You should improve oui-angular instead of suggesting html. The value of using components is undeniable.

Reply:
Hello,

I think that this syntax seems to be debating:

```html
<oui-back-button data-href="{{ ctrl.$state.href('^') }}">
```

I think we should propose something different and considering that href attribute is not a string by default, but a value which can be interpreted by angular directly (href vs ngHref style ^^).

Example:

```html
<oui-back-button data-href="ctrl.$state.href('^')">
<oui-back-button data-href="'/example/of/path'"
```

Instead of:

```html
<oui-back-button data-href="{{ ctrl.$state.href('^') }}">
<oui-back-button data-href="/example/of/path">
```

IMO, i think it's strange to get the mustache syntax in attribute :/

I can take a lot of examples of librairies which follow this pattern like angular-translate. Currently, we do a lot of:

```html
<a title="{{ 'my_translation' | translate }}">
```

Instead of using angular-translate correctly, something like:

```html
<a data-translate-attr-title="my_translation">
```

The second syntax is more readable I think. I should be cool to keep this syntax for ui-kit :)

What do you think about that? cc @AxelPeter

Reply:
We're working on the integration of ui-router in ui-angular. Once it will be done there would have no more problem with this. (ETA 12days)

Reply:
I totally agree with your comments where components offer **an undeniable values**.

However, implementing components not in the right way (where we have to update it later due to the `sref` example here) is not a comfortable option in my point of view.

Thanks @whispyy for your inputs.

Reply:
@jleveugle 

`IMO, i think it's strange to get the mustache syntax in attribute :/`

In the same way as having single quote between double quote :)
But I agree in the way that we have to use/think more our components around `angular-translate` and `ui-outer`. There are analysis tasks on these subjects, any feedbacks are appreciated.

@antleblanc A more appropriate way is to update the component, not using LESS because the component doesn't fulfill our needs :)

In a general way, [our components sttatus](https://master.ui-kit.ovh/#!/documentation/component-status) are driven like this:

- beta: is a proposal and need feedbacks before integration
- rc: has not be integrated, or is currently tested in our projects
- complete: has been integrated in minimum 2 of our projects and passed code reviews

Reply:
@jleveugle "href vs ngHref" ? Both take mustaches ;)